web/ui/react-app/src/pages/alerts/CollapsibleAlertPanel.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC, useState, Fragment } from 'react';
+import { Link } from '@reach/router';
+import { Alert, Collapse, Table, Badge } from 'reactstrap';
+import { Rule, RuleStatus } from './AlertContents';
+
+interface CollapsibleAlertPanelProps {
+  rule: Rule;
+  showAnnotations: boolean;
+}
+
+export const alertColors: RuleStatus<string> = {
```

Comment:
Does this need to be exported? I only see it used in this file.

Reply:
Nope. Will remove the export, thanks!